tests/config/index.test.ts

```diff
@@ -0,0 +1,39 @@
+import { runInValidEnvironment, isEnable } from '../../src/config'
+
+describe('Expresso', () => {
```

Comment:
Describe should make a ref to at least "config"

Reply:
Done